### ISSUE TITLE
DataStreamingClient error when subscribing for multiple tickers minute data via WebService 

### ISSUE THREAD

**Issue by Tifoid** [alpacahq/alpaca-trade-api-csharp#385]
The following code results in an error in my DataStreamingClient's OnError handler of Error: 400 with a Message: "Invalid Syntax" whenever I attempt to register for more than one ticker (symbol). 

```cs               
var minBarSubscription = _broker.DataStreamingClient.GetMinuteBarSubscription(symbol);

var result = _broker.DataStreamingClient.SubscribeAsync(minBarSubscription, stoppingToken);

barSubscriptionDictionary.Add(symbol, minBarSubscription);
_logger.LogInformation("Subscribed for minute bars for symbol: " + symbol);

minBarSubscription.Received += (IBar agg) =>
{
    processingMethod(translateBarFormat(agg), stoppingToken);
};
_logger.LogInformation("Processing delegate established for minute bars for symbol: " + symbol);
```

I invoke the above code multiple times in succession to register for each of the tickers (symbols).

This error seems to be introduced with version 5.1.1.

**Comment by OlegRa** [comment]
@Tifoid I've reproduced this problem but I still need time to find the real root cause, fixing it, and releasing the updated version.

In the meantime, you can try the next possible workarounds:

1. Collect all subscription objects and subscribe all symbols at once
2. Await each `SubscribeAsync` call and add a delay after completion

**Comment by Tifoid** [comment]
@OlegRa 
Just wanted to pass along some more information.  After changing my code to build an array of Subscriptions and then performing the SubscribeAsync (asynchronously) I am now able to get data.  I did see some potential odd behavior when registering for multiple tickers in that when I registered for a single ticker I immediately started getting minute data. When I registered two tickers i didnt see data for a good 5 minute (at which time I gave up).  Admittedly, these tickers have low volume, but I was watching them via yahoo finance and I was seeing changing minute data on the one ticker.  Finally, I subscribed for 50 tickers including SPY as the first ticker in the list.  With this configuration I was getting a large number of minute data elements back across multiple tickers (most notably SPY) every minute.

While I cant say there is something amiss (need more testing), there may be some sort of problem where if the first ticker in the list has a "low minute change rate" it might hold up the other tickers from reporting data via the subscription.  I'll attempt to do more testing the next time I am home when the market is open ... but that wont be for a couple of days.
